acquia/acquia_cms
# ACMS-717: Fix error, media item with UUID 'e4dda7f7-ec92-4aaf-97aa-04…

**Motivation**


Fixes #NNN
When enabling starter module we noticed error in the log:
"During rendering of embedded media: the media item with UUID "e4dda7f7-ec92-4aaf-97aa-04c0ecbc415c" does not exist."
**Proposed changes**

This changes default content's content
**Alternatives considered**


**Testing steps**

[Visit here](http://orionacmsdev.prod.acquia-sites.com/article/news/article-one-long-length-wrapping-placeholder-heading)
and verify you can see video embedded on the page.
**Merge requirements**
- [x] _Bug_
